### PR TITLE
Update ako-operator for 1.6.0 release

### PR DESCRIPTION
**Pull request by sbhatp**
Signed-off-by: Sharath Bhat <sharathb@vmware.com>

## What this PR does / why we need it
This PR updates ako-operator addon for 1.6.0 release

## Details for the Release Notes (PLEASE PROVIDE)

```release-note
1.6.0 release folder for ako-operator
```

## Which issue(s) this PR fixes

Fixes: #

### CODE REVIEW

File: addons/packages/ako-operator/1.6.0/bundle/.imgpkg/images.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: imgpkg.carvel.dev/v1alpha1
+images:
+- annotations:
+    kbld.carvel.dev/id: projects-stg.registry.vmware.com/tkg/ako-operator:v1.5.0_vmware.1
```
Comment (HanFa) [inline addons/packages/ako-operator/1.6.0/bundle/.imgpkg/images.yml:5]: this should be changed later (in another PR), after we have formal 1.6.0 ako-o build

**Review by HanFa — DISMISSED** [review]
LGTM, there's only one place in readme needs to be updated

File: addons/packages/ako-operator/1.6.0/README.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+tanzu package available list ako-operator.tanzu.vmware.com -n tkg-system
+```
+
+The ako-operator package with specified version and values can be installed with the Tanzu package CLI.
+Refer to [next section](#configuration-values) for a detailed list of configuration values in `values.yaml`.
+
+```shell
+tanzu package install ako-operator --values-file values.yaml \
+  --package-name ako-operator.community.tanzu.vmware.com \
+  --version 1.5.0 --namespace tkg-system
```
Comment (XudongLiuHarold) [inline addons/packages/ako-operator/1.6.0/README.md:43]: This one also need to change to 1.6.0

File: addons/packages/ako-operator/1.6.0/README.md [hunk 3]
```diff
@@ -0,0 +1,214 @@
+# Load Balancer Operator
+
+This package reconciles Cluster API objects and manages the lifecycles of [AVI Kubernetes Operator (AKO)](https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/tree/release-1.5.2) using [ako-operator](https://github.com/vmware-samples/load-balancer-operator-for-kubernetes).
```
[the remark this answers is not in the document]
Comment (sbhatp) [reply addons/packages/ako-operator/1.6.0/README.md]: Fixed

File: addons/packages/ako-operator/1.6.0/README.md (hunk 2 above)
Comment (sbhatp) [reply addons/packages/ako-operator/1.6.0/README.md]: Fixed

File: addons/packages/ako-operator/1.6.0/bundle/.imgpkg/images.yml (hunk 1 above)
Comment (sbhatp) [reply addons/packages/ako-operator/1.6.0/bundle/.imgpkg/images.yml:5]: ok

**Review by XudongLiuHarold — APPROVED** [review]

**Pull request merged**